Cargo.lock

```diff
@@ -642,7 +642,7 @@ dependencies = [
 
 [[package]]
 name = "rubyfmt-main"
-version = "0.8.0-pre"
+version = "0.8.1"
```

Comment:
I think the lockfile didn't get updated during the last version bump, so I included it here 🤷‍♂️